packages/clay-color-picker/src/Basic.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Splotch from './Splotch';
+
+Basic.propTypes = {
```

Comment:
If you add babel-plugin-transform-class-properties (`@babel/plugin-proposal-class-properties` in Babel v7) then you can replace these with `static` property declarations, which might read a bit nicer. (Longer term, adopting Flow or TypeScript would obviate the need for propType declarations entirely).

Reply:
Since `Basic` is a function component I was unable to add it as a static property. If it was a class I would totally be with you on that.

Reply:
Ah, sorry. In my first read-through I noticed you doing this on the `GradientSelector` class and then I came back and put my comment on the first instance of `propTypes` that I found, where it doesn't actually apply.